README.md

```diff
@@ -502,6 +502,47 @@ as p-values and z-statistics. When the CATE model is linear and parametric, then
   
   </details>
   
+
+### Policy Learning
+
+You can also perform direct policy learning from observational data, using the doubly robust method for offline
+policy learning. These methods directly predict a recommended treatment, without internally fitting an explicit
+model of the conditional average treatment effect.
+
+<details>
+  <summary>Doubly Robust Policy Learning (click to expand)</summary>
+
+```Python
+from econml.policy import DRPolicyTree, DRPolicyForest
+from sklearn.ensemble import RandomForestRegressor
+
+# fit a single binary decision tree policy
+policy = DRPolicyTree(max_depth=1, min_impurity_decrease=0.01, honest=True)
```

Comment:
not sure. migth be reasonable. Was more showing some capabilities of what is parameterizable

Reply:
If the defaults are reasonable, and these arguments are also reasonable, and there's no particular reason to assume one is better than the other, then that's fine.

Otherwise, I'd try to make sure that whatever you put here will be a good start for someone to copy and paste, rather than show off functionality that's more advanced but requires more knowledge to use well.

Reply:
yeap I think this is good and along these lines